ci/emscripten-entry.sh

```diff
@@ -7,6 +7,6 @@ source /emsdk-portable/emsdk_env.sh &> /dev/null
 
 # emsdk-portable provides a node binary, but we need version 8 to run wasm
 # NOTE: Do not forget to sync Node.js version with `emscripten.sh`!
-export PATH="/node-v14.17.0-linux-x64/bin:$PATH"
+export PATH="/node-v16.20.0-linux-x64/bin:$PATH"
```

Comment:
This change can be omitted after PR #3321 lands.

Reply:
Thanks! Adjusted